wordpress-mobile/WordPress-iOS
# iOS 14 Stats Widgets - Rename target and bundle id

Renamed target, bundle id, folder and supporting files.
Swift files and code renaming will be handled in a separate PR.

At this stage, it will run on a simulator, not yet on a device.

Fixes #NA

To test:
- build/run on simulator (iOS 14) and make sure you can install the iOS 14 Today Widget

PR submission checklist:
